package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Aug 18 08:52:41 UTC 2016 - mfilka@suse.com
+
+- bnc#991486
+  - extended fix for this bug from 3.1.163. Do not report invalid
+    names incorrectly
```

Comment:
This description means that there is an invalid name and the report that we produce about it is incorrect. It has completely misled me about the actual nature of the bugfix.

IIUC the real bug is that we report "Configuration name is invalid" even if there is no name to edit. So what about:
> Do not report "Configuration name is invalid"  inNetworkCard Setup / Hardware when there is no name to edit.